docs/src/developer_folder/instrument_plugins.rst

```diff
@@ -248,7 +248,7 @@ and further processed by DAQ_Scan or DAQ_Viewer instances. The code below is an
 
 .. code-block:: python
 
-    from pymodaq.utils.data import Axis, DataFromPlugins, DataToExport
+    from pymodaq_data.data import Axis, DataFromPlugins, DataToExport
```

Comment:
no here we should leave as it was, because DataFromPlugins is only defined in pymodaq not in pymodaq_data

Reply:
does it makes sense to write as follow ?
```
from pymodaq_data.data import Axis, DataToExport
from pymodaq.utils.data import DataFromPlugins
```

Reply:
I am not sure i pushed this modification yet